test/agentchat/test_groupchat.py

```diff
@@ -1,5 +1,15 @@
 import pytest
 import autogen
+import threading  # So that the function call runs for limited time
```

Comment:
where is this used?

Reply:
I have removed threading -- Apologies as I was testing the feasibility of constraining the run-away group chat to 5 seconds but failed to do it with threading. It is not used anymore.